pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -64,6 +69,53 @@ const (
 	remotePowerShellCmdPrefix = "powershell.exe -NonInteractive -ExecutionPolicy Bypass "
 )
 
+// FileInfo contains information about a file
+type FileInfo struct {
+	Path   string
+	SHA256 string
+}
+
+// NewFileInfo returns a pointer to a FileInfo object created from the specified file
+func NewFileInfo(path string) (*FileInfo, error) {
+	contents, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, errors.Wrapf(err, "could not get contents of file")
+	}
+	return &FileInfo{
+		Path:   path,
+		SHA256: fmt.Sprintf("%x", sha256.Sum256(contents)),
+	}, nil
+}
+
+// filesToTransfer is a map of what files should be copied to the Windows VM and where they should be copied to
+var filesToTransfer map[*FileInfo]string
+
+func init() {
+	filesToTransfer = make(map[*FileInfo]string)
+	srcDestPairs := map[string]string{
+		wkl.IgnoreWgetPowerShellPath: remoteDir,
+		wkl.WmcbPath:                 k8sDir,
+		wkl.HybridOverlayPath:        k8sDir,
+		wkl.HNSPSModule:              remoteDir,
+		wkl.FlannelCNIPluginPath:     cniDir,
+		wkl.WinBridgeCNIPlugin:       cniDir,
+		wkl.WindowsExporterPath:      k8sDir,
+		wkl.HostLocalCNIPlugin:       cniDir,
+		wkl.WinOverlayCNIPlugin:      cniDir,
+		wkl.KubeProxyPath:            k8sDir,
+		wkl.KubeletPath:              k8sDir,
+	}
+	for src, dest := range srcDestPairs {
+		f, err := NewFileInfo(src)
+		if err != nil {
+			log.Error(err, "could not create FileInfo object", "file", src)
+			os.Exit(1)
+		}
+		filesToTransfer[f] = dest
+	}
+
+}
+
```

Comment:
I suggest moving this into a separate package. Maybe combine it with `wellknownlocations.go`  and change the package name to something like `files`.